docs/.htmltest.yml

```diff
@@ -2,7 +2,7 @@
 CheckDoctype: false
 IgnoreDirs:
   - favicons
-  - docs/crd-ref/lifecycle/
+  - docs/api-ref/lifecycle/
```

Comment:
please keep both paths in here. since we have the older versions of the docs with the old folder, we wanna ignore both paths